packages/core/register/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@parcel/register",
+  "version": "2.0.0",
+  "main": "./lib/register.js",
+  "license": "MIT",
+  "scripts": {
+    "test": "echo this package has no tests yet",
+    "test-ci": "yarn build && yarn test",
+    "format": "prettier --write \"./{src,bin,test}/**/*.{js,json,md}\"",
+    "lint": "eslint . && prettier \"./{src,bin,test}/**/*.{js,json,md}\" --list-different",
```

Comment:
eslint and prettier are run from the root by the way, so it's not totally necessary to define package scripts for them

Reply:
Thanks, I've already removed these in the next commit :)